docs/high_level_functions.rst

```diff
@@ -0,0 +1,6 @@
+
+High Level Functions
+====================
+
+.. automodule:: pyrefinebio.high_level_functions
+   :members:
```

Comment:
Is there any way to get some spacing between the functions? This seems cramped and it's hard to scan the different functions to find the one I want:

![image](https://user-images.githubusercontent.com/4026833/104374085-57176500-54ef-11eb-9d26-abcf1ef46436.png)